README.md

```diff
@@ -88,6 +89,25 @@ atlas init-app -name=my-application -db
 # specifies a docker registry
 atlas init-app -name=my-application -registry=infoblox
 ```
+
+```sh
+# generates an application with additional services
+# the input file (expand.txt in the example below) 
+#   is a list of strings (letters only), one service name on each line
+atlas init-app -name=my-application -expand=expand.txt -db=true
+```
+
+```sh
+# example `expand.txt` file for use with -expand option
+# Each line must either be a single string of letters only, 
+# or two strings separated by a single comma. The latter option
+# Is for the use case where the user wants a customized pluralization
+# of a word. In the example below, "Artifacts" will pluralize to 
+# "Artifactss", and "Kubernetes" will pluralize to "KubeCluster"
+
+Artifacts
+Kubernetes,KubeCluster
+```
```

Comment:
Might merit a comment that the objects should probably be specified as singular objects so that the pluralizations are natural as table/service names?

Reply:
yes, good point. Definitely ambiguous there given the example has a plural name.